src/core/lib/channel/channel_stack_builder.h

```diff
@@ -95,6 +95,10 @@ bool grpc_channel_stack_builder_move_next(
 bool grpc_channel_stack_builder_move_prev(
     grpc_channel_stack_builder_iterator *iterator);
 
+/// Return an iterator at \a filter_name, or NULL if not found.
```

Comment:
To make this interface work more like the C++ STL iterator interface, would it make sense to have this return an iterator pointing to the end instead of NULL if the name is not found?